omnigibson/robots/manipulation_robot.py

```diff
@@ -417,7 +417,7 @@ def _add_arm_control_dict(self, fcns, arm):
         eef_link_idx = self._articulation_view.get_body_index(self.eef_links[arm].body_name)
         fcns[f"eef_{arm}_jacobian_relative"] = lambda: ControllableObjectViewAPI.get_relative_jacobian(
             self.articulation_root_path
-        )[eef_link_idx, :, start_idx : start_idx + self.n_joints]
+        )[-(self.n_links - eef_link_idx), :, start_idx : start_idx + self.n_joints]
```

Comment:
sorry can you add a comment explaining why the following line?
```
-(self.n_links - eef_link_idx)
```

Reply:
done!